jetbrains-core/resources/icons/resources/sqs/SqsQueue.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="17px" viewBox="0 0 18 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Amazon simple queue service (SQS) - light</title>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Amazon-simple-queue-service-(SQS)---light" transform="translate(0.866390, 0.254418)" fill="#999999" fill-rule="nonzero">
```

Comment:
I always get this confused but this is backwards. the 999999 one should be `_dark` one.